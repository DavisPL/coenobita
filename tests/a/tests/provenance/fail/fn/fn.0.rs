#![allow(unused_variables)]
#![allow(unused_assignments)]

#[cnbt::provenance((*,*) fn((b,b)) -> (bin,bin))]
fn id(x: i32) -> i32 {
    x
}

fn main() {
    #[cnbt::provenance((b,b))]
    let x = id(b::value());
}