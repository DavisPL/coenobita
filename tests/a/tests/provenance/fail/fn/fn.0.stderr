error: expected (bin,bin), found (b,bin)
 --> tests/provenance/fail/fn/fn.0.rs:6:5
  |
6 |     x
  |     ^

error: expected (b,b), found (b,bin)
  --> tests/provenance/fail/fn/fn.0.rs:11:16
   |
11 |     let x = id(b::value());
   |                ^^^^^^^^^^
